alarm/wiringpi/PKGBUILD

```diff
@@ -5,10 +5,10 @@ buildarch=20
 
 _pkgname='wiringPi'
 pkgname=wiringpi
-pkgver=97.78b5c32
+pkgver=100.b0a60c3
 pkgrel=1
 pkgdesc="WiringPi: An implementation of most of the Arduino Wiring functions for the Raspberry Pi"
-arch=('arm')
+arch=('arm' 'arm7h')
```

Comment:
I tried to rebuild the package on my raspi3 and it gave me an error, that the architecture is not supported for some reason. Thatswhy I added arm7h (arm7 hardfloat). No Idea how you could build it without this. You can build the package for arm6h though and run it on arm7, but then its slower.